signingscript/script.py

```diff
@@ -73,78 +56,24 @@ def write_json(self, *args):
     log.info("Done!")
 
 
-# config {{{1
-def get_default_config(base_dir=None):
-    """Create the default config to work from.
+def _set_signing_context(context):
+    def _stubbed_out_method(*args):
+        pass
 
-    Args:
-        base_dir (str, optional): the directory above the `work_dir` and `artifact_dir`.
-            If None, use `..`  Defaults to None.
+    context.write_json = _stubbed_out_method
 
-    Returns:
-        dict: the default configuration dict.
+    if not hasattr(context, 'signing_servers'):
+        context.signing_servers = None
 
-    """
-    base_dir = base_dir or os.path.dirname(os.getcwd())
-    default_config = {
```

Comment:
This is useful to have, for local testing. Is there any reason we stripped it out?

Reply:
I agree, this can be useful. I wasn't sure how used it was. This was also an issue when loading the whole config in scriptworker.

I have a cleaner fix in mind. I'll implement the first part in scriptworker